.github/workflows/release.yml

```diff
@@ -1,54 +1,200 @@
-name: release
+name: release-v2
```

Comment:
Any specific reason for this?

Reply:
Nop, a remaining of the ci test on my branch ahah, let me remove that